operations/products.md

```diff
@@ -29,351 +32,361 @@ Returns all products offered together with the specified services. Note this ope
+| `CreatedUtc` | string | required | Creation date and time of the product in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the product in UTC timezone in ISO 8601 format. |
+| `ConsumptionMoment` | [Product consumption moment](products.md#product-consumption-moment) | required | Consumption moment of the product. |
+| ~~`IsDefault`~~ | ~~boolean~~ | ~~required~~ | **Deprecated!** |
+| ~~`Name`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product. **Deprecated!** Please use Names~~ **Deprecated!** Use `Names` instead.|
+| ~~`ExternalName`~~ | ~~string~~ | ~~optional~~ | ~~Name of the product meant to be displayed to customer. **Deprecated!** Please use ExternalNames~~ **Deprecated!** Use `ExternalNames` instead.|
+| ~~`ShortName`~~ | ~~string~~ | ~~optional~~ | ~~Short name of the product. **Deprecated!** Please use ShortNames~~ **Deprecated!** Use `ShortNames` instead.|
+| ~~`Description`~~ | ~~string~~ | ~~optional~~ | ~~Description of the product. **Deprecated!** Please use Descriptions~~ **Deprecated!** Use `Descriptions` instead.|
+| ~~`Charging`~~ | ~~[Product charging mode](products.md#product-charging-mode)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Posting`~~ | ~~[Product posting](products.md#product-posting)~~ | ~~required~~ | **Deprecated!** |
+| ~~`Pricing`~~ | ~~[Product pricing](products.md#product-pricing)~~ | ~~required~~ | **Deprecated!** Use `Price` instead.|
+
+#### Product charging mode
+
+* `Once`
+* `PerTimeUnit`
+* `PerPersonPerTimeUnit`
+* `PerPerson`
+
+#### Product posting
+
+* `Once`
+* `Daily`
+
+#### Product posting mode
+
+* `Once`
+* `PerTimeUnit`
+
+#### Product options
+Options of the product.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `BillAsPackage` | boolean | required | Product should be displayed as part of a package. |
+
+#### Product classifications
+Classifications of the service.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Food` | boolean | required | Product is classified as food. |
+| `Beverage` | boolean | required | Product is classified as beverage. |
+| `Wellness` | boolean | required | Product is classified as wellness. |
+| `CityTax` | boolean | required | Product is classified as city tax. |
+
+#### Product pricing
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Discriminator` | [Product pricing data discriminator](products.md#product-pricing-data-discriminator) | required | Determines type of value. |
+| `Value` | object | required | Structure of object depends on `Discriminator`. |
+
+#### Product pricing data discriminator
+
+* `Absolute`
+* `Relative`
+
+#### Relative product price
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Multiplier` | number | required |  |
+| `Target` | string | optional |  |
+| `TaxRateCodes` | array of string | optional |  |
+| `ProductIds` | array of string | optional |  |
```

Comment:
Missing descriptions

Reply:
Yeah, these are objects that weren't in the docs, the whole object is deprecated... don't want to spend much time on these deprecated fields as discussed when we started with those😢 

@jonas-paul @jnv do you think would be OK to slap deprecated on all those or dig into the code how the properties behave and describe them? 🤔 We could also remove it from the docs but it's a response and if I remember correctly, the agreement was that we present everything in the response and just slap deprecated on it or am I wrong?

Reply:
I'm ok with marking all of the properties deprecated as long as the type does not have other (non deprecated) usages.

Reply:
Only usage is here, so we are fine

Reply:
Agreed, let's mark it as deprecated. I don't even see it linked from schema above, is it possible the property linking it is hidden?

Reply:
It's coproduct 😢 is there a way to link it to correct objects?